server/src/test/resources/db.properties

```
@@ -17,7 +17,7 @@
 
 
 # management server clustering parameters, change cluster.node.IP to the machine IP address
-# in which the management server is running
+# in which the management server is running
 cluster.node.IP=127.0.0.1
 cluster.servlet.port=9090

```

are these carriage returns intentionally left in?
```suggestion
cluster.node.IP=127.0.0.1
cluster.servlet.port=9090
```

---

I found an issue thread here https://github.com/pre-commit/pre-commit-hooks/issues/196
which talks about the 50/50 rule and how to fix the line endings locally.

So that file had mostly Windows style carriage returns so did not get fixed at first.  
Now I have just targeted Java and Python files for this initial first run clean up.